### ISSUE TITLE
ipdatabase.com has shutdown

### ISSUE THREAD

**Issue by rodmatos** [tevslin/cloudflarepycli#9]
Apparently, ipdatabase.com has shutdown and the domain is up for sale.
This leads to the library failing due to the response sent by the webserver. https://github.com/tevslin/cloudflarepycli/blob/29ea4530ac8bf227c4090237ea2f91b8b4730d45/src/cloudflarepycli/cloudflareclass.py#L81-L88

```bash
$ curl -L https://ipdatabase.com/ip/ip_address
<html>
  <head>
    <meta http-equiv='refresh' content='0; url=https://ipdatabase.com/' />
  </head>
</html>
```

I wonder if it would be possible to remove the dependencies from external services, since Cloudflare should provide all the relevant information.

**Comment by ccmpbll** [comment]
@rodmatos I used this in one of my own projects and noticed this a few days ago. I have created a pull request that removes the dependency on ipdatabase.com that should resolve the problem. 

@tevslin could you please review this [PR](https://github.com/tevslin/cloudflarepycli/pull/10) and let us know you can approve?

**Comment by tevslin** [comment]
closed with version 1.8. removed dependency and added the ability to get the data from cloudflare. Thank you
